code/world.py

```diff
@@ -16,7 +16,7 @@
 os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
 args = parse_args()
 
-ROOT_PATH = "/Users/gus/Desktop/light-gcn"
+ROOT_PATH = os.getcwd()
```

Comment:
I recommend obtaining the root path from the `__file__` variables, something like
```python
ROOT_PATH = os.path.dirname(os.path.dirname(__file__))
```